lib/ecto/schema.ex

```diff
@@ -255,6 +255,8 @@ defmodule Ecto.Schema do
   * `__schema__(:primary_key)` - Returns a list of primary key fields (empty if there is none);
 
   * `__schema__(:fields)` - Returns a list of all non-virtual field names;
+  * `__schema__(:db_fields)` - Returns a list of all field names as stored in db;
+  * `__schema__(:aliases)` - Returns a list of fields with their changed source;
```

Comment:
I agree with @raarts. `field_source` is probably better to account for NoSQL dbs which don't have columns in the strict sense.